packages/mobile/e2e/scripts/check-e2e-wallet-balance.ts

```diff
@@ -0,0 +1,7 @@
+import { E2E_TEST_WALLET } from './consts'
+import { balanceError, getBalance } from './utils'
+;(async () => {
+  console.log(E2E_TEST_WALLET)
+  console.table(await getBalance())
+  await balanceError()
```

Comment:
On the first look I was surprised, does it store the balance in a global variable? What does `balanceError` mean?

It's better to explicitly pass the address being checked.

I propose to change it to:
```suggestion
  await checkBalance(E2E_TEST_WALLET)
```

`checkBalance` would display and check the balance.

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/2325/commits/2a72d8204532a885b1fb40b95a45f1c7ce9f3fef! I kept checkBalance and getBalance separate as we don't need to call checkBalance on the faucet.